src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -128,7 +133,8 @@ class HoldRequest extends React.Component {
           name="delivery-location"
           id="available-electronic-delivery"
           value="edd"
-          onChange={this.onRadioSelect}
+          checked={this.state.checkedLocNum === -1}
+          onChange={(e) => { this.onRadioSelect(e, -1); }}
```

Comment:
You can also do without the inner `{ ... }` and do `onChange={(e) => this.onRadioSelect(e, -1)}`, but that's up to you.